src/Core/Models/Api/Request/SendRequestModel.cs

```diff
@@ -13,7 +13,7 @@ namespace Bit.Core.Models.Api
     public class SendRequestModel
     {
         public SendType Type { get; set; }
-        public long? FileLength { get; set; }
+        public long? FileLength { get; set; } = null;
```

Comment:
Is the default assignment of `null` necessary since `default(Nullable<long>)` already equals `null`? Not sure if you were running into serialization issues here or not to necessitate the assignment.

Reply:
I didn't see any issues, and you're right I don't think it's needed. I just wanted to make it explicit